test/cli/runtime_types/test.out

```diff
@@ -9,7 +9,7 @@ test/cli/runtime_types/test.rb:11: Expected `T.class_of(A)` but found `Runtime o
     test/cli/runtime_types/test.rb:6:
      6 |X = T.type_alias {A}
         ^
-  Note:
+  Detailed explanation:
```

Comment:
This is intentional because we're not using an ad-hoc note, we're using the first-class error details mechanism.